bin/gwc/gwc.ml

```diff
@@ -153,9 +155,27 @@ let errmsg =
   \  object files end with .gwo\n\
    and [options] are:"
 
+(* same function in bin/setup/setup.ml *)
```

Comment:
maybe we should put it in lib/ then, I'm not sure where

Reply:
Tell me where and I will do it

Reply:
thanks!